src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Model/Column.cs

```diff
@@ -472,6 +471,49 @@ public override void Set(SqlDataRecord record, int ordinal, byte[] value)
         }
     }
 
+    public class BinaryColumn : Column<byte[]>
+    {
+        public BinaryColumn(string name, int length)
+            : base(name, SqlDbType.Binary, true, length)
```

Comment:
@johnstairs I've hard coded `Nullable = true` here. Is there a way to collect this info from the SQL file and pass it in to the Column constructor?

Reply:
DId you look at `SqlVisitor.IsColumnNullable`?

Reply:
As discussed, I hard coded `Nullable = false` in 613031e, since the rowversion value can never be `null`.